torch/csrc/jit/frontend/sugared_value.cpp

```diff
@@ -151,6 +151,10 @@ std::shared_ptr<SugaredValue> SimpleValue::attr(
     }
   } else if (auto classType = value_->type()->cast<ClassType>()) {
     // This is a class, emit the proper attribute lookup
+    if (auto overloaded_methods = classType->findOverloadedMethod(field)) {
+      return std::make_shared<MethodValue>(
+          getValue(), overloaded_methods.value());
```

Comment:
Nit:  constructor of `MethodValue` takes argument by value, not by reference. Consider using `std::move` on `overloaded_methods.value()` to save one copy.